cirq/study/trial_result.py

```diff
@@ -266,11 +267,17 @@ def histogram(  # type: ignore
             keys=[key], fold_func=lambda e: fold_func(e.iloc[0]))
 
     def __repr__(self):
-        return ('cirq.TrialResult(params={!r}, '
-                'repetitions={!r}, '
-                'measurements={!r})').format(self.params,
-                                             self.repetitions,
-                                             self.measurements)
+        def item_repr(entry):
+            key, val = entry
+            return '{!r}: {}'.format(key, proper_repr(val))
+
+        measurement_dict_repr = (
+                '{' +
+                ', '.join([item_repr(e) for e in self.measurements.items()]) +
+                '}')
+
+        return 'cirq.TrialResult(params={!r}, measurements={})'.format(
+            self.params, measurement_dict_repr)
```

Comment:
What does this new repr look like?

Reply:
```
cirq.TrialResult(params=cirq.ParamResolver({'a': 2}), measurements={'xy': np.array([[1, 0], [0, 1]])})
```

Reply:
The main criteria of a repr is that it round trips, so I'm not too worried about the details. It happens to look pretty good.